lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -257,6 +263,7 @@ def test_course_end_date_for_non_certificate_eligible_mode(self):
             block.description,
             'After this date, course content will be archived.'
         )
+        self.assertEqual(block.title, 'Course End')
```

Comment:
^^ Actually, on review, it's pretty fleshed out -- it doesn't test the actual rendering of the date, but that's covered by previous tests (does this render, etc.). I *think* it's okay as is.